app/src/main/java/org/wikipedia/settings/DeveloperSettingsActivity.kt

```diff
@@ -0,0 +1,18 @@
+package org.wikipedia.settings
+
+import android.content.Context
+import android.content.Intent
+import org.wikipedia.activity.SingleFragmentActivity
+
+class DeveloperSettingsActivity : SingleFragmentActivity<DeveloperSettingsFragment>() {
+    public override fun createFragment(): DeveloperSettingsFragment {
+        return DeveloperSettingsFragment.newInstance()
+    }
+
+    companion object {
+        @JvmStatic
+        fun newIntent(context: Context?): Intent {
```

Comment:
Any reason for `context` to be nullable?

Reply:
No. Done.